tests/hardware_tests/test_gaze_env.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import os
+from typing import Dict, List
+
+import numpy as np
+import pytest
+from spot_rl.envs.gaze_env import (
+    GazeController,
+    construct_config_for_gaze,
+    update_config_for_multiple_gaze,
+)
+from spot_wrapper.spot import Spot
+
+hardware_tests_dir = os.path.dirname(os.path.abspath(__file__))
+test_configs_dir = os.path.join(hardware_tests_dir, "configs")
+test_data_dir = os.path.join(hardware_tests_dir, "data")
+test_nav_trajectories_dir = os.path.join(test_data_dir, "nav_trajectories")
+test_square_nav_trajectories_dir = os.path.join(
+    test_nav_trajectories_dir, "square_of_side_200cm"
```

Comment:
same

Reply:
thanks for pointing this out. It is not needed.
will remove it.

Reply:
addressed